free-1.md

```diff
@@ -7,44 +7,168 @@ good way to meet their needs.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
+experience like now? What would their experience be like if you could help
 them?_
 
+For live theaterical productions, there exists a diconnect between the language
+with which one describes actor and scenic movements through space and the
+lighting that illuminates that space. Currently, lighting designers will watch
+a run through of the show to note where different scenic elements and actors are
+over the duration of the production. They will then envision lighting cues&mdash;
+shifts in the lighting&mdash;and generally describe what is happening along with
+a verbal line or action that prompts the change. The lighting designer then
+looks at the their light plot&mdash;a technical drawing where each lighting
+instrument will be hung (and aimed) in the venue&mdash;and spits off a string
+of channel numbers (or group names) to a light board operator along with a fade
+time and intensity level; they are essentially translating their more
+descriptive cues to a less descriptive computer form. Optionally, when
+programming in the light cue which list channel numbers and intensities, the
+designer *might* instruct the board operator to include a note next to the cue
+specifying (in some abstract sense) what the cue is meant to achieve. When the
+designer plays back the cues with the action on stage they look along on their
+decriptive cue sheet and the computer version and call out more channel numbers
+and intensities to edit the cues. During a show where the stage manager is
+telling a board operator when to hit `GO` for the next cue does not necessarily
+have a clear picture of what the cue is going to do besides taking notes and
+memorizing it. This can be problematic if a cue is skipped or someone is not
+in a place they are supposed to be&mdash;it makes it very difficult for the
+stage manager to decide when to skip cues in these cases unless they are intimitely
+familiar with each and every cue.
```

Comment:
This sounds super complicated - I certainly think it is ripe for a better solution, but it seems like there already exists a DSL. I haven't read the rest of your writeup yet, but I'm curious to see how you improve on the existing system